package.json

```diff
@@ -17,68 +17,62 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "dependencies": {
-    "chalk": "^4.1.2",
-    "lit-element": "^2.5.1"
+    "@aurodesignsystem/auro-icon": "^4.0.2",
```

Comment:
Also move the auro-icon to dependencies since it is in the src file.
Let me know if this is purposely not needed, @blackfalcon.

Reply:
Please move that to `peerDependencies`

Reply:
done~